File: examples/python/input_output.fr.py
/* {"title":"input/output","platform":"python","tags":["python"]} */
print('Entrez un mot et un nombre séparés par un espace :')
inputStr = input().split(' ')
if len(inputStr) == 2:
    str = inputStr[0]
    num = int(inputStr[1])

    print('Longueur du mot * nombre =', len(str) * int(num))
else:
    print('Entrée incorrecte')

File: examples/c/input_output.fr.c
/* {"title":"entrée/sortie","mode":"unix","tags":["plain"]} */
#include <stdio.h>
unsigned long strlen(const char * s) {
  unsigned long l = 0;
  while (*s++) ++l;
  return l;
}
int main() {
    int a, n;
    char s[12];
    printf("Entrez un mot et un nombre:\n");
    n = scanf("%s %d", s, &a);
    if (n == 2) {
        printf("Longueur du mot * nombre = %lu\n", strlen(s) * a);
    } else {
        printf("Pas de valeur!\n");
    }
    return 0;
}